bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersTracker.java

```diff
@@ -59,19 +60,30 @@ static String setReviewersMarker(int numReviewers, String role) {
                     break;
                 }
             } else {
-                updatedLimits.replace(r, updatedLimits.get(r) + remainingAdditional);
+                if (remainingAdditional > updatedLimits.get(r)) {
+                    // Set the number for the lower roles to remove.
+                    remainingRemovals = remainingAdditional - updatedLimits.get(r);
+                    // The new number of the reviewers should larger or equal than the requirement of the '.jcheck/conf' file.
```

Comment:
I would suggest moving this comment to before line 63 and express it something like this:

// The new value cannot the lower than the value in .jcheck/conf

Reply:
Fixed